gridscale/resource_gridscale_k8s.go

```diff
@@ -154,6 +179,11 @@ func resourceGridscaleK8s() *schema.Resource {
 							Description: "Storage type.",
 							Required:    true,
 						},
+						"surge_node_count": {
```

Comment:
In the panel, surge_node is enabled/disabled. We do not use surge_node_count> 1. I would suggest making the surge_node as in the panel behavior. Further, it would be great if the surge_node feature implemented in a different PR.